sdk/runanywhere-kotlin/src/jvmAndroidMain/kotlin/com/runanywhere/sdk/public/extensions/RunAnywhere+VLM.jvmAndroid.kt

```
@@ -0,0 +1,289 @@
+        imagePath = image.filePath,
+        imageData = image.pixelData,
+        imageBase64 = image.base64Data,
+        imageWidth = image.width,
+        imageHeight = image.height,
+        prompt = prompt,
+        optionsJson = optionsJson,
+    )
+
+    vlmLogger.info(
+        "VLM processing complete: ${cppResult.completionTokens} tokens in ${cppResult.totalTimeMs}ms " +
+            "(${String.format("%.1f", cppResult.tokensPerSecond)} tok/s)",
```

detekt (1.23.8)

[warning] String.format("%.1f", cppResult.tokensPerSecond) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)